hkmc2/shared/src/test/mlscript/codegen/TraceLogIndent.mls

```diff
@@ -0,0 +1,92 @@
+:js
+
+fun plus(x, y) = x + y
+
+:traceJS
+fun g(z) =
+  if
+    z < -1 then g(z + 1)
+    z == -1 then 42
+    z == 0 then plus(0)
+    z > 0 then g(z - 1)
+
+fun f(x, y)(z) =
+  x + y + g(z)
+
+:traceJS
+fun h(x) =
+  f(1,2)(x)
+
+:traceJS
+:re
+h(3)
+//│ ═══[RUNTIME ERROR] Error: Function 'plus' expected 2 arguments but got 1
```

Comment:
Any idea why we're not seeing the trace up to the runtime error?

Reply:
Yes, because when the ReplHost encounters an error, only the error messages are kept in its replies (https://github.com/hkust-taco/mlscript/blob/99863022a7a9089e162a8d8387c55c17a931daf7/hkmc2/jvm/src/test/scala/hkmc2/ReplHost.scala#L197-L205), maybe I can also make a patch to this in this PR?

Reply:
Sure, that would be great, if you don't mind.

Reply:
Thanks! I have pushed a commit to address this.

Reply:
Great!

One last tweak, though: can we get the runtime error message after the output? Currently, it appears out of logical order:

```
//│ ═══[RUNTIME ERROR] Error: Function 'plus' expected 2 arguments but got 1
//│ CALL h(3)
//│ | CALL g(3)
//│ | | CALL g(2)
//│ | | | CALL g(1)
//│ | | | | CALL g(0)
```

It should be:


```
//│ CALL h(3)
//│ | CALL g(3)
//│ | | CALL g(2)
//│ | | | CALL g(1)
//│ | | | | CALL g(0)
//│ ═══[RUNTIME ERROR] Error: Function 'plus' expected 2 arguments but got 1
```

Reply:
Thanks, I have pushed the change. I also added the missing ">" which start each line of the output, so that now it's:
```
//│ > CALL h(3)
//│ > | CALL g(3)
//│ > | | CALL g(2)
//│ > | | | CALL g(1)
//│ > | | | | CALL g(0)
//│ ═══[RUNTIME ERROR] Error: Function 'plus' expected 2 arguments but got 1
```